### PR TITLE
Added parsing of Screenshot name

### PR DESCRIPTION
**Pull request by vborodav**
Added new model CMAttachment to parse screenshot name straight from attachment to avoid issue with different extensions.

### CODE REVIEW

**Comment by vborodav** [comment]
@Krivoblotsky please take a look)

File: XCSummary/Templates/CMHTMLReportBuilder.m [hunk 1]
```diff
@@ -137,12 +137,13 @@ - (void)_appendActivity:(CMActivitySummary *)activity indentation:(CGFloat)inden
     if (activity.hasScreenshotData)
     {
         templateFormat = [self _decodeTemplateWithName:ActivityTemplateWithImage];
-        NSString *imageName = [NSString stringWithFormat:@"Screenshot_%@.png", activity.uuid.UUIDString];
+        CMAttachment *attachment = [activity.attachments firstObject];
```
Comment (Krivoblotsky) [inline XCSummary/Templates/CMHTMLReportBuilder.m]: Is it ok to take only first attachment? What if an activity has more than one attachment?

Comment (vborodav) [reply XCSummary/Templates/CMHTMLReportBuilder.m]: Well yeah, I thought about that.

We need to modify template to be able to add more than one screenshot, or even a file.

Comment (Krivoblotsky) [reply XCSummary/Templates/CMHTMLReportBuilder.m]: Got it, ok then.
One more question. `firstObject` is nullable, let's check for not nil and return early in this case.

**Comment by vborodav** [comment]
@Krivoblotsky added check if attachment is nil

**Review by Krivoblotsky — APPROVED** [review]

**Pull request merged**